site/website/replace-version-config.js

```diff
@@ -0,0 +1,17 @@
+const replace = require('replace-in-file');
+const options = {
+  files: "../docs/**/**.md",
+  from: [/{jet-version}/g, /{imdg-version}/g],
```

Comment:
isn't `{` a reserved character in regex? This matches three a's:
```regex
a{3}
```

Reply:
I know that is but in this format without escaping it works as expected.